### ISSUE TITLE
Use official dark/light visuals as org banner instead of custom-made image

### ISSUE THREAD

**Issue by benoit74** [offspot/.github#4]
Currently the org banner at https://github.com/offspot/.github/blob/main/profile/README.md is a custom-made image which is not part of the official [visuals](https://github.com/offspot/overview/wiki/Visuals).

Since Github is perfectly supporting dark/light picture in markdown for some time, we should probably give it a try to use official logos:

```
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="./dark.png">
  <img alt="Text changing depending on mode. Light: 'So light!' Dark: 'So dark!'" src="./light.png">
</picture>
```

Or we could even directly use the SVG which embeds this dark/light code.

Or do I miss something?

**Comment by rgaudin** [comment]
You're right. Not sure if GH supports the themed SVG in README (it doesn't in issues) but otherwise using both PNGs as suggested would work.
